core/assets/bundles/bundle_ru.properties

```diff
@@ -871,7 +871,7 @@ unit.eclipse.name = Затмение
 unit.mono.name = Моно
 unit.poly.name = Поли
 unit.mega.name = Мега
-unit.risse.name = Раскол
+unit.risse.name = Риссо
```

Comment:
This should be *unit.risso.name*, I changed it.

Reply:
https://github.com/Anuken/Mindustry/pull/2233/commits/da4f042fa4c4ecc8386c2dd991178cffc2c8c0fc